INSTALL.rst

```diff
@@ -40,7 +47,7 @@ Disable SELinux
 ---------------
 
 The setup described below runs into problems with SELinux related to the sqlite
-database. For now the recommended solution is to simply disable SELinux. In
+database and apache. For now the recommended solution is to simply disable SELinux. In
```

Comment:
Quoting apoorvemohan (2015-06-10 09:58:59)

>    If a user is not using SQLite he may skip this step

We hit this with something when using postgres as well, though it wasn't 
clear exactly what the issue was. It doesn't seem to work without 
disabling selinux regardless?